attachments/07_image_views.cpp

```diff
@@ -228,14 +289,14 @@ class HelloTriangleApplication {
         if (capabilities.currentExtent.width != std::numeric_limits<uint32_t>::max()) {
```

Comment:
Maybe implement `chooseSwapPresentMode` the same as in `06_swap_chain_creation.cpp`?